thread/CMakeLists.txt

```diff
@@ -293,6 +293,14 @@ add_library(j9thr_obj OBJECT
 	ut_j9thr.c
 )
 
+if(OMR_WARNINGS_AS_ERRORS)
+	target_compile_options(j9thr_obj PRIVATE ${OMR_WARNING_AS_ERROR_FLAG})
+endif()
+
+if(OMR_ENHANCED_WARNINGS)
+	target_compile_options(j9thr_obj PRIVATE ${OMR_ENHANCED_WARNING_FLAG})
+endif()
+
```

Comment:
Do we want to remove the setting of the option from below that was setting it on the library?

Reply:
Sure, I can remove it.

Reply:
Seeing as the j9thrstatic doesn't have any objects itself, adding compilation flags to it doesn't make sense.

Reply:
done